styles/global.css

```diff
@@ -31,5 +31,37 @@ header .logo {
 }
 
 main {
-    padding: 20px;
+    height: 900px;
+    flex-direction: column;
```

Comment:
the flex-direction property is used when using the display: flex; property. here you shouldn't really need flex-direction: column because you're using display: grid instead